exporters/trace/instana/README.md

```diff
@@ -0,0 +1,73 @@
+# OpenCensus Instana Trace Exporter
+[![Build Status][travis-image]][travis-url]
+[![Windows Build Status][appveyor-image]][appveyor-url]
+[![Maven Central][maven-image]][maven-url]
+
+The *OpenCensus Instana Trace Exporter* is a trace exporter that exports
+data to Instana. [Instana](http://www.instana.com/) Instana is a distributed
+tracing system.
+
+## Quickstart
+
+### Prerequisites
+
+[Instana](http://www.instana.com/) forwards traces exported by applications
+instrumented with Census to its backend using the Instana agent processes as proxy.
+If the agent is used on the same host as Census, please take care to deactivate
+automatic tracing.
+
+
+### Hello Stan
+
+#### Add the dependencies to your project
```

Comment:
You can leave this section empty for now since it's not released to maven yet.

Reply:
I do not follow this conclusion. I assume that after its merged, it will be eventually released and then users will want to use it via maven? why should I leave this out and add this later?

Reply:
Because v0.11.1 is our current version (already released), and does not have artifact of this exporter. Users might be confused when they read README, since opencensus-exporter-trace-instana is not included in io.opencensus v0.11.1.

You could use v0.12.0 instead, or you could leave a TODO here and we'll update this section once we've made a new release.

Reply:
Alright, then ill use 0.12 - do you have a procedure in place to update versions in readmes on release?

Reply:
Yes, please add this README to RELEASING.md: https://github.com/census-instrumentation/opencensus-java/blob/master/RELEASING.md#update-release-versions-in-documentations-and-build-files. We'll update versions for all README file when we made new releases.

Reply:
Cool will do. The way the sed works would give preference to using 0.1.1.1 right now because the sed would catch it. I guess thats a first time integration issue that one has to take care for the first release.